core/core.js

```diff
@@ -124,6 +124,9 @@
     opSync("op_close", rid);
   }
 
+  // Provide bootstrap namespace
+  window.__bootstrap = {};
```

Comment:
It would be cool if somehow we could delete __bootstrap in core after all the extensions have been loaded. That would remove the need for the caller to delete something provided by core.

Reply:
If we convert everything to extensions and don't use `__bootstrap` outside of extensions then it's very easy to do, but for the moment `runtime` / `cli` ops are still registered imperatively (will convert them in a follow up PR)